src/com/kennymaness/Character.java

```diff
@@ -1,125 +1,129 @@
+        this.characterClass = characterClass;
+    }
 
-        public String getCharacterClass() {return characterClass;}
+    public String getCharacterClass() {return characterClass;}
 
-        // returns the ones in blue, made into lowercase
-        public static ArrayList<String> allClasses() {
-            ArrayList<String> classes = new ArrayList<String>();
-            for (CharacterClass c : CharacterClass.values()) {
-                classes.add(c.toString().toLowerCase());
-            }
-            return classes;
+    // returns the ones in blue, made into lowercase
+    public static ArrayList<String> allClasses() {
+        ArrayList<String> classes = new ArrayList<String>();
+        for (CharacterClass c : CharacterClass.values()) {
+            classes.add(c.toString().toLowerCase());
         }
+        return classes;
+    }
 
-        // returns the ones in green, made into lowercase
-        public static ArrayList<String> allClassValues() {
-            ArrayList<String> classes = new ArrayList<String >();
-            for (CharacterClass c : CharacterClass.values()) {
-                classes.add(c.toString().toLowerCase());
-            }
-            return classes;
+    // returns the ones in green, made into lowercase
+    public static ArrayList<String> allClassValues() {
+        ArrayList<String> classes = new ArrayList<String >();
+        for (CharacterClass c : CharacterClass.values()) {
+            classes.add(c.toString().toLowerCase());
         }
+        return classes;
     }
+}
+
+
+public class Character {
+
+    static final int maxHitPoints = 20;
 
     // declare the attributes!
+    private String characterName = "";
     private int currentHitPoints;
-    CharacterRace characterRace;
-    static CharacterClass characterClass;
-    private String characterName;
     private ArrayList<Weapon> weaponPouch;
-    Boolean isArmed;
+    private ArrayList<Item> itemPouch;
     private int[] abilityScores;
+    CharacterClass characterClass;
+    CharacterRace characterRace;
+    Boolean isArmed;
 
     // constructor
     public Character(int currentHitPoints,
-                     CharacterRace characterRace,
-                     CharacterClass characterClass,
-                     String characterName,
                      ArrayList<Weapon> weaponPouch,
-                     Boolean isArmed,
-                     int[] abilityScores) {
+                     ArrayList<Item> itemPouch,
+                     int[] abilityScores,
+                     CharacterClass characterClass,
+                     CharacterRace characterRace,
+                     Boolean isArmed) {
 
         this.currentHitPoints = currentHitPoints;
-        this.characterRace = characterRace;
-        Character.characterClass = characterClass;
-        this.characterName = characterName;
         this.weaponPouch = weaponPouch;
-        this.isArmed = isArmed;
+        this.itemPouch = itemPouch;
```

Comment:
Added an itemPouch to the Character class that defaults to an empty array on creation (just like weaponPouch)